app/src/pages/inside/profilePage/localizationBlock/localizationBlock.jsx

```diff
@@ -78,6 +78,16 @@ const messages = defineMessages({
   },
 });
 
+const langName = {
+  en: 'english',
+  uk: 'ukrainian',
+  ru: 'russian',
+  be: 'belarusian',
+  zh: 'chinese',
+};
+
+const langNameByCode = (code) => langName[code];
```

Comment:
Do we really need that function?
We can easily langName[lang].
And maybe LangNameMap will be a better name?

Reply:
I think this function in 1 line improves the readability of the code (_langNameByCode(lang) more understandable than langName[lang]_), and the name langNameByCode makes it clear what the function does, unlike LangNameMap